### PR TITLE
fix unit test after replacing MutationObserver with ResizeObserver

### PR DESCRIPTION
**Pull request by mdulat**
No longer need isOwnMutation because callback is triggered only on resize. Unit test was failing because appending child directly to content instead of p.parentNode was not triggering slotchange event.

### CODE REVIEW

File: d2l-more-less.html [hunk 1]
```diff
@@ -318,14 +318,7 @@
...
 
-		__reactToChanges: function moreLessReactToChanges(mutations) {
-			if (mutations
-				&& Array.isArray(mutations)
-				&& mutations.every(this.__isOwnMutation.bind(this))
-			) {
-				return;
-			}
-
+		__reactToChanges: function moreLessReactToChanges(entries) {
```
Comment (omsmith) [inline d2l-more-less.html]: What's `entries`?

Comment (mdulat) [reply d2l-more-less.html]: It's an array of objects passed in by ResizeObserver. Each one contains the target and contentRect (width, height, top, left) of item that has been resized. I guess we don't use this data when we handle the resize.

**Comment by dlockhart** [comment]
I'm still seeing test failures when I run `npm test` and when I visit the tests locally in my browser by running `polymer serve` and visiting the test page.

There are 4 different scenarios and only one of them is passing:
- Polymer 1 + shady DOM: fail (`http://localhost:8081/components/d2l-more-less/test/d2l-more-less.html?wc-shadydom=true&wc-ce=true`)
- Polymer 1 + shadow DOM: pass (`http://localhost:8081/components/d2l-more-less/test/d2l-more-less.html?dom=shadow`)
- Polymer 2 + shady DOM: fail (`http://localhost:8001/components/d2l-more-less/test/d2l-more-less.html?wc-shadydom=true&wc-ce=true`)
- Polymer 2 + shadow DOM: fail (`http://localhost:8001/components/d2l-more-less/test/d2l-more-less.html?dom=shadow`)

I think you may need to put the original mutation observer back and somehow re-integrate the resize observer in...

**Comment by mdulat** [comment]
Using both observers the unit tests, demo, and our usage in activity feed are all working.

File: d2l-more-less.html [hunk 2]
```diff
@@ -126,9 +128,13 @@
...
+			if (this.__resizeObserver) {
+				this.__resizeObserver.disconnect();
+				this.__resizeObserver = null;
+			}
+			if (this.__mutationObserver) {
+				this.__mutationObserver.disconnect();
+				this.__mutationObserver = null;
 			}
 
 			this.__content.removeEventListener('load', this.__bound_reactToChanges, true);
```
Comment (omsmith) [inline d2l-more-less.html]: Need a `this.__bound_reactToMutationChanges = null;` around here

**Pull request merged**